example/example/models.py

```diff
@@ -7,5 +7,5 @@
 
 class Page(Route):
     """A route that renders raw HTML."""
-    raw_html = models.TextField()
+    raw_html = models.TextField(verbose_name='Raw HTML')
```

Comment:
Should this be translatable?

Reply:
Ah shoot. good spot.

Reply:
Oh, come to think of it, I think this is ok because it's in the example.